people/handlers_test.go

```diff
@@ -263,14 +302,56 @@ type dummyService struct {
 	wg           *sync.WaitGroup
 }
 
-func (s *dummyService) getPeople() ([]personLink, bool) {
+func (s *dummyService) getPeopleOld() ([]personLink, bool) {
```

Comment:
Was going to keep it as got a bad feeling that style of "links" might be used by another service. Will rename it to something more sensible.